requirements/base.txt

```diff
@@ -12,6 +12,7 @@ whitenoise==6.4.0
 xmltodict==0.13.0
 zeep==4.2.1
 urllib3==2.0.3
+redis==5.0.0
```

Comment:
I'm unsure if this should only be in `prod.txt`, during development we don't need it as we want always_eager anyway. What do you think @hklarner @m4ra

Reply:
makes sense. i'll change it.